ls1intum/bamboo-server-notification-plugin
# Add dependency for parser project

### Checklist

### Motivation and Context


The parser functionality for static code analysis reports has been published to maven central. The parsing functionality can therefore be deleted from the notification plugin.

### Description


- Add dependency for parser project to pom
- Delete parser functionality
- Bump number of last logs from 1000 to 5000

### Steps for Testing


1. Deploy the plugin on a local Bamboo installation
2. Test programming exercises with and without static code analysis
3. Let builds fail and check whether the compilation error is displayed
